x/clock/client/cli/tx.go

```diff
@@ -0,0 +1,129 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+
+	"github.com/CosmosContracts/juno/v18/x/clock/types"
+)
+
+// NewTxCmd returns a root CLI command handler for certain modules/Clock
+// transaction commands.
+func NewTxCmd() *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Clock subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	txCmd.AddCommand(
+		NewRegisterClockContract(),
+		NewUnregisterClockContract(),
+		NewUnjailClockContract(),
+	)
+	return txCmd
+}
+
+// NewRegisterClockContract returns a CLI command handler for registering a
+// contract for the clock module.
+func NewRegisterClockContract() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "register [contract_bech32]",
+		Short: "Register a contract for clock. Sender must be admin of the contract.",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			senderAddress := cliCtx.GetFromAddress()
+			contractAddress := args[0]
+
+			msg := &types.MsgRegisterClockContract{
+				SenderAddress:   senderAddress.String(),
+				ContractAddress: contractAddress,
+			}
+
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(cliCtx, cmd.Flags(), msg)
+		},
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+	return cmd
+}
+
+// NewUnregisterClockContract returns a CLI command handler for unregistering a
+// contract for the clock module.
+func NewUnregisterClockContract() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "unregister [contract_bech32]",
+		Short: "Unregister a contract for clock. Sender must be admin of the contract.",
```

Comment:
`Sender must be admin of the contract.` should be in the `Long:` cobra.Command section instead for all cmds

(`Short:` is shown in the CLI when you type in the parent ex: `junod tx clock` while `Long:` is shown when you request specific help ex: `junod tx clock mycmd --help`)

Reply:
Thank you, updated with recent commit